components/search/stories/template.js

```diff
@@ -41,13 +41,12 @@ export const Template = ({
 			ActionButton({
 				iconName: "Search",
 				isDisabled,
+				isHovered,
 				size,
 				isFocused: !isDisabled && (isFocused || isKeyboardFocused),
 				isQuiet: true,
 				customClasses: [
 					`${rootClass}-actionButton`,
-					isHovered && "is-hover",
-					isDisabled && "is-disabled",
```

Comment:
These were evaluating to `"false"` previously:
<img width="632" height="50" alt="Screenshot 2025-08-20 at 10 55 29 AM" src="https://github.com/user-attachments/assets/c1f667a2-9da1-407c-a7d4-5ead1676fd3a" />